src/routes/safe/components/Balances/SendModal/screens/AddressBookInput/index.jsx

```diff
@@ -77,6 +80,16 @@ const AddressBookInput = ({
       if (isCustomTx && isValidText === undefined) {
         isValidText = await mustBeEthereumContractAddress(resolvedAddress)
       }
+
+      // Filters the entries based on the input of the user
+      const filteredADBK = addressBook.filter(adbkEntry => {
+        const { name, address } = adbkEntry
+        return (
+          name.toLowerCase().includes(addressValue.toLowerCase()) ||
+          address.includes(addressValue)
```

Comment:
why not use the same strategy as with the name? to make both `address` values lowerCased?

Reply:
Done